demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -0,0 +1,104 @@
+    type: Optional[str] = None
+    hidden: Optional[bool] = None
+    name_xsoar: Optional[str] = Field(None, alias="name:xsoar")
+    name_marketplacev2: Optional[str] = Field(None, alias="name:marketplacev2")
+    name_xpanse: Optional[str] = Field(None, alias="name:xpanse")
+    name_xsoar_saas: Optional[str] = Field(None, alias="name:xsoar_saas")
+    name_xsoar_on_prem: Optional[str] = Field(None, alias="name:xsoar_on_prem")
+    required_xsoar: Optional[bool] = Field(None, alias="required:xsoar")
+    required_marketplacev2: Optional[bool] = Field(None, alias="required:marketplacev2")
+    required_xsoar_saas: Optional[bool] = Field(None, alias="required:xsoar_saas")
+    required_xsoar_on_prem: Optional[bool] = Field(None, alias="required:xsoar_on_prem")
+    description_xsoar: Optional[str] = Field(None, alias="description:xsoar")
+    description_marketplace_v2: Optional[str] = Field(None, alias="description:marketplacev2")
+    description_xpanse: Optional[str] = Field(None, alias="description:xpanse")
+    description_xsoar_saas: Optional[str] = Field(None, alias="description:xsoar_saas")
+    description_xsoar_on_prem: Optional[str] = Field(None, alias="description:xsoar_on_prem")
+    default_value_xsoar: Optional[Any] = Field(None, alias="defaultValue:xsoar")
+    default_value_marketplace_v2: Optional[Any] = Field(None, alias="defaultValue:marketplacev2")
+    default_value_xpanse: Optional[Any] = Field(None, alias="defaultValue:xpanse")
+    default_value_xsoar_saas: Optional[Any] = Field(None, alias="defaultValue:xsoar_saas")
+    default_value_xsoar_on_prem: Optional[Any] = Field(None, alias="defaultValue:xsoar_on_prem")
+    deprecated_xsoar: Optional[bool] = Field(None, alias="deprecated:xsoar")
+    deprecated_marketplace_v2: Optional[bool] = Field(None, alias="deprecated:marketplacev2")
+    deprecated_xpanse: Optional[bool] = Field(None, alias="deprecated:xpanse")
+    deprecated_xsoar_saas: Optional[bool] = Field(None, alias="deprecated:xsoar_saas")
+    deprecated_xsoar_on_prem: Optional[bool] = Field(None, alias="deprecated:xsoar_on_prem")
+
+
+class Output(BaseStrictModel):
+    content_path: Optional[str] = Field(None, alias="contentPath")
+    context_path: Optional[str] = Field(None, alias="contextPath")
+    description: str
+    type: Optional[str] = None
+
+
+class Important(BaseStrictModel):
+    context_path: str = Field(..., alias="contextPath")
+    description: str
+    related: Optional[str] = None
+    description_xsoar: Optional[str] = Field(None, alias="contextPath")
+    description_marketplacev2: Optional[str] = Field(None, alias="description:marketplacev2")
+    description_xpanse: Optional[str] = Field(None, alias="description:xpanse")
+    description_xsoar_saas: Optional[str] = Field(None, alias="description:xsoar_saas")
+    description_xsoar_on_prem: Optional[str] = Field(None, alias="description:xsoar_on_prem")
+
+
+class ScriptType(StrEnum):
+    TYPE_PWSH = "powershell"
```

Comment:
do we have a value for this one under `constants.py`? If we do, it's best to reuse it from there